application/models/services/QuestionAggregateService/SubQuestionsService.php

```diff
@@ -162,6 +162,9 @@ private function storeSubquestion(
                 $subquestion = DI::getContainer()->make(Question::class);
             }
         }
+        if (empty($subquestion)) {
```

Comment:
I think we must avoid this before ?

No ?

Throw an exception is a really good idea :)

Reply:
> I think we must avoid this before 

Not sure there are lot of places before where to move this... do you find a better spot?

Reply:
> > I think we must avoid this before
> 
> Not sure there are lot of places before where to move this... do you find a better spot?

Sorry, I didn't make myself clear :)

I mean : call SubQuestionsService with invalid values for sub qids must be checked before :) and have an error management before.

Here : seems a good place inside service, but my opinion : this is more a error before (in controller) : value send to service must be clean.